.github/workflows/build_and_test_debug.yml

```diff
@@ -74,13 +74,10 @@ jobs:
 
   windows_debug_build:
     name: Windows Debug Build
-    runs-on: windows-2019
+    runs-on: ubuntu-20.04
     timeout-minutes: 10
```

Comment:
*nit* we might need to increase this timeout because we've experienced timeouts when building Windows in the past. it's OK to increase it in another PR.